### PR TITLE
migrate FAQ

### PR DESCRIPTION
**Pull request by morloy**

### CODE REVIEW

File: src/markdown/faq.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+Go to the *Transactions* page and in the *Filter* field, type *Convertible loan*.
+
+Or open the *Round Modeling* on the *Cap Table* page, click *Add* and *Remaining Convertibles*. This also allows you to simulate the conversion of the loans (see next point).
+
+##### How do I convert my convertible loans?
+
+There are two ways of converting convertible loans:
+
+1.  Through the round modeling tool, which also [calculates for you](https://blog.ledgy.com/convertible-loans-7948385a5d66) how many shares should be issued for the convertible, taking cap, discount and interest into account. Go to the *Cap Table* page and click on *Round Modeling*. Existing convertibles should already be displayed. If not, add them by clicking on *Add *and then *Remaining Convertibles.*
```
Comment (bbinoy) [inline src/markdown/faq.md]:
```suggestion
1.  Through the round modeling tool, which also [calculates for you](https://blog.ledgy.com/convertible-loans-7948385a5d66) how many shares should be issued for the convertible, taking cap, discount and interest into account. Go to the *Cap Table* page and click on *Round Modeling*. Existing convertibles should already be displayed. If not, add them by clicking on *Add* and then *Remaining Convertibles.*
```

File: src/markdown/faq.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+Go on the *Shareholders* page, add you investor’s email address (at this point the shareholder does NOT receive any email). Then just click the invite button on the right-hand side. You can choose three access right levels.
+
+1. **Personal** The investor will be able see her own stake in her [portfolio](https://blog.ledgy.com/ledgy-launches-investor-dashboard-a1ad1a95df07).
+2. **View-only** View everything, but not able to edit.
+3. **Admin** Full access.
+
+##### How can I enable share numbers?
+
+Go to your company’s *Settings* page, click on the edit icon in the *Company Profile* box, then activate the *Share Numbers* toggle. You can also check out our opinionated [blog post](https://blog.ledgy.com/are-numbered-shares-necessary-15c27f3457a8) about the necessity of numbered shares.
```
Comment (bbinoy) [inline src/markdown/faq.md]:
```suggestion
Go to your company’s *Settings* page and under your company details activate the *Share Numbers* toggle. Nice side-effect: This automatically adds consistent share numbers to your current transactions. It always takes the lowest available range of numbers for the next transaction.
```

File: src/markdown/faq.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+1.  Through the round modeling tool, which also [calculates for you](https://blog.ledgy.com/convertible-loans-7948385a5d66) how many shares should be issued for the convertible, taking cap, discount and interest into account. Go to the *Cap Table* page and click on *Round Modeling*. Existing convertibles should already be displayed. If not, add them by clicking on *Add *and then *Remaining Convertibles.*
+2.  Find your convertible loan on the *Transactions* page. Then click on the menu (dots) in the transaction itself and select *Convert*. You need to calculate the number of shares yourself.
+
+##### In which browser does the app work best?
+
+The app works best in Google Chrome and Mozilla Firefox. If don’t use one of these browsers and notice a weird behavior, please let us know.
+
+##### How can I create shareholder groups?
+
+Go to the *Stakeholders* page. Hover over the shareholder and you’ll see the edit field for the shareholder’s group. By default the groups *Founders*, *Employees* and *Investors* already exist. To add a new group, simply start typing a name in the field and click on *Create new group*.
```
Comment (bbinoy) [inline src/markdown/faq.md]:
```suggestion
Go to the *Stakeholders* page, click the three dots next one, click *Edit* and type the group in the respective field. By default, the groups *Founders*, *Employees* and *Investors* already exist. To add a new group, simply start typing a name in the field and click on *Create new group*.
```

File: src/markdown/help/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+##### Add new transactions
+
+- Got to the transactions page.
+- Click on _Add Transactions_ and choose the transaction you’d like to add. Fill in the required details and click on save.
+- Once the transaction appears in the transactions list, you can also __attach the corresponding documents__ such as a legal contract:
+
+![](attach-documents.png)
+
+- Once you are done with adding all new transactions click on the _Apply_ button (read on to find out why).
```
Comment (bbinoy) [inline src/markdown/help/getting-started.md]:
```suggestion
- Once you are done with adding all new transactions click on the _Commit_ button (read on to find out why).
```

File: src/markdown/help/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+##### Commit your draft
+
+All the new transactions you’ve added will be saved as your new draft. Once you are done with adding them and are happy with it, you can hit the _Commit_ button to save your draft as your official transactions list. Don’t worry, you will be prompted to review your changes before confirming.
+
+###### What does the _Commit_ button do?
+
+Two things:
```
Comment (bbinoy) [inline src/markdown/help/getting-started.md]: A third thing would be 

_Saving a pdf snapshot of your new cap table and transactions. This creates an audit-trail for you for future due-diligences. The snapshot is even certified by the [blockchain notary](https://ledgy.com/help/faq). Of course you can delete this document if you don't want it._

**Comment by morloy** [comment]
> Why are there 2 files now? They are diverging already, so we should decide which one is the correct one.
> 
> * src/markdown/faq.md
> * src/markdown/help/faq.md
> 
> Ich denke es wäre auch gut, zu erwähnen _how to get started with Employee Participation Plan Management_

In `/help/faq.md`. Da sollten sie auch bleiben.

File: src/markdown/help/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- Click on _Add Transactions_ and choose the transaction you’d like to add. Fill in the required details and click on save.
+- Once the transaction appears in the transactions list, you can also __attach the corresponding documents__ such as a legal contract:
+
+![](attach-documents.png)
+
+- Once you are done with adding all new transactions, click on the _Commit_ button (read on to find out why).
+
+##### Commit your draft
+
+All the new transactions you’ve added will be saved as your new draft. Once you are done with adding them and are happy with it, you can hit the _Commit_ button to save your draft as your official transactions list. Don’t worry; you will be prompted to review your changes before confirming.
```
Comment (bbinoy) [inline src/markdown/help/getting-started.md]:
```suggestion
All the new transactions you’ve added will be saved as your new draft. New or modified transactions are marked in gray. The question-mark below the transaction number tells you what has changed. 

Once you are done with adding and editing transactions, you can hit the _Commit_ button to save your draft as your new official transactions list. Don’t worry; you will be prompted to review your changes before confirming and can still make changes to your transactions after committing.
```

**Pull request merged**